packages/medusa/src/strategies/__tests__/price-selection.js

```diff
@@ -333,23 +333,6 @@ const toTest = [
       },
     },
   ],
-  [
```

Comment:
self-review: Those test are never called in develop and the error does not exists